src/gucs.rs

```diff
@@ -65,3 +74,7 @@ pub(crate) fn tracing_level() -> tracing::Level {
         .map(|v| v.parse().expect("plrust.tracing_level was invalid"))
         .unwrap_or(tracing::Level::INFO)
 }
+
+pub(crate) fn auto_recompile() -> bool {
+    PLRUST_AUTO_RECOMPILE.get()
+}
```

Comment:
This does no computational effort and thus does not need a wrapper function like this, simply use `gucs::PLRUST_AUTO_RECOMPILE.get()` elsewhere.